tectonic/dpx-dpxfile.c

```diff
@@ -19,7 +19,9 @@
 */
 
 #include <time.h>
+#if HAVE_UNISTD_H
```

Comment:
Use `#ifdef` instead. It is not done everywhere but we should avoid adding more.